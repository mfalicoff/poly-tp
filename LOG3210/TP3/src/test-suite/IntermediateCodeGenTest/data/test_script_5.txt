Declaration
num a;
EndDeclaration

setup {
  a = - 4;
}

code {
  a = a + 1;
}

teardown {
  a = 0;
}

setup()
code()
code()
teardown()
