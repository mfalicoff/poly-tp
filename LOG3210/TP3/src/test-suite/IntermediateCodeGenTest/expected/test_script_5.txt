_t0 = - 4
a = _t0
_t1 = a + 1
a = _t1
_t2 = a + 1
a = _t2
a = 0
_L0
